client/src/components/exercise/lab7/components/File.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { AI_CORRECT, LOCKED_FILE } from "../../../../constants/lab7";
+import LOCKED from "../../../../assets/images/lab7/lock.png";
+import OPEN from "../../../../assets/images/lab7/unlock.png";
+
+const File = ({ data }) => {
+  const image = data.decision === LOCKED_FILE ? LOCKED : OPEN;
+  const alt = `A .png image of ${
+    data.decision === LOCKED_FILE ? "a locked" : "an unlocked"
+  } lock.`;
+  const reportClassName =
+    data.report === AI_CORRECT ? "tw-text-[#2e8540]" : "tw-text-[#e31c3d]";
+
+  return (
+    <div className={"tw-flex tw-flex-col tw-items-center"}>
+      <div
+        className={"tw-space-y-3 tw-flex tw-flex-col tw-justify-center file"}
+      >
+        {data.report !== undefined && (
+          <div>
+            <img className={"tw-h-14 tw-w-14"} src={image} alt={alt} />
+            <p className={"tw-mt-1.5 tw-font-bold"}>{data.decision} FILE</p>
+          </div>
+        )}
+        <div>
+          <p className={"tw-leading-tight tw-font-bold"}>{data.fileName}</p>
```

Comment:
this is just my opinion but I don't think filename is too important in this context and especially when there's a lot of text, maybe this doesn't have to be bold?

Reply:
Good point, removed the bold for the filename.